.packaging/Dockerfile.c_env

```diff
@@ -0,0 +1 @@
+Dockerfile.cpp_env
```

Comment:
I don't think C and C++ need a separate container. It's not even possible to build `clang` with support for only one or the other and system packages between them will probably have a large amount of overlap.

Reply:
C is only a symbolic link to the C++ container, but just having it in there for continuity's sake s.t. every language has its own container..